docs/content/dagster-cloud/developing-testing/managing-deployments.mdx

```diff
@@ -0,0 +1,193 @@
+2. Access the **Deployments** page using on eof the following options:
+   - Click the **deployment switcher > View all deployments**.
+   - Click **your user icon > Cloud Settings**.
+3. Click the **+ New deployment** button.
+4. In the modal that displays, fill in the following:
+   - **Name** - Enter a name for the deployment.
+   - **Initial deployment permissions** - Select the permissions you want to use to create the deployment:
+     - **Empty permissions** - Creates the deployment with an empty set of permissions. **Note**: Only Organization Admins will be able to manage the deployment until other uses are granted Admin or Editor permissions.
+     - **Copy from** - Creates the deployment using permissions duplicated from an existing deployment.
+5. When finished, click **Create deployment**.
+
+---
+
+## Deleting deployments
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users#understanding-user-permissions">
+    Organization Admin permissions
+  </a>{" "}
+  are required to delete deployments. Additionally, note that deleting a
+  deployment also deletes all its associated data, including code locations,
+  jobs, schedules, and sensors.
+</Note>
+
+To delete a deployment:
+
+1. Sign in to your Dagster Cloud account.
+2. Access the **Deployments** page using on eof the following options:
+   - Click the **deployment switcher > View all deployments**.
+   - Click **your user icon > Cloud Settings**.
+3. Click the **Delete** button next to the deployment you want to delete.
+4. When prompted, confirm the deletion.
+
+---
+
+## Configuring deployment settings
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users#understanding-user-permissions">
+    Organization Admin permissions
+  </a>{" "}
+  are required to modify deployment settings.
+</Note>
+
+Deployment settings can be configured in the Dagster Cloud interface or using the dagster-cloud CLI. Refer to the [deployment settings reference](/dagster-cloud/developing-testing/deployment-settings-reference) for more info about individual settings.
+
+<TabGroup>
+  <TabItem name="Dagster Cloud">
+   To configure deployment settings in the Dagster Cloud UI:
+
+   <ol>
+    <li>Sign in to your Dagster Cloud account.</li>
+    <li>Access the <strong>Deployments</strong> page using one of the following:
+     <ul>
+      <li>Click the <strong>deployment switcher > View all deployments</strong>.</li>
+      <li>Click <strong>your user icon > Cloud Settings</strong>.</li>
+     </ul>
+    </li>
+    <li>Click the <strong>Settings</strong> button next to the deployment you want to configure.</li>
+    <li>In the window that displays, configure settings for the deployment.</li>
+    <li>When finished, click <strong>Save deployment settings</strong>.</li>
+   </ol>
+  </TabItem>
+  <TabItem name="dagster-cloud CLI">
+<Note>
+<strong>Note:</strong> <code>dagster-cloud</code> 0.13.14 or later must be installed to run the CLI. Agent and/or job code doesn't need to be upgraded.
```

Comment:
this version is so old at this point that I wonder if we even need to say this

Reply:
I'm honestly not sure. Is there a way we can tell if anyone is still using it?